modules/virt-what-you-can-do-with-virt.adoc

```diff
@@ -21,5 +21,8 @@ custom resources to enable virtualization tasks. These tasks include:
 An enhanced web console provides a graphical portal to manage these virtualized
 resources alongside the {product-title} cluster containers and infrastructure.
 
+{VirtProductName} is tested with OpenShift Container Storage (OCS) and designed to use with 
+OCS features for the best experience.
```

Comment:
Consider replacing "best experience" with a concrete, positive benefit of using OCS.